src/components/molecules/page-header/page-header.md

```diff
@@ -24,3 +24,33 @@
   }}
 />
 ```
+
+```js
+<PageHeader
+  title="Clients"
+  description={{
+    text: 'Setup a mobile, web or IoT application to use Auth0 for Authentication.',
+    learnMore: '/clients'
+  }}
+  primaryAction={{
+    label: 'Create Client',
+    icon: 'plus',
+    handler: () => {}
+  }}
+/>
+```
+
```

Comment:
Let's add the following paragraph:

A Page Header can have a single secondary action:

Reply:
Done!